src/main/scala/com/github/swagger/scala/converter/SwaggerScalaModelConverter.scala

```diff
@@ -61,8 +61,8 @@ class SwaggerScalaModelConverter extends ModelResolver(Json.mapper()) {
 
   private def matchScalaPrimitives(`type`: AnnotatedType, nullableClass: Class[_]): Option[Schema[_]] = {
     val annotations = Option(`type`.getCtxAnnotations).map(_.toSeq).getOrElse(Seq.empty)
-    annotations.collectFirst { case ann: JsonScalaEnumeration => ann } match {
-      case Some(enumAnnotation) => {
+    annotations.headOption match {
+      case Some(enumAnnotation: JsonScalaEnumeration) => {
```

Comment:
I'm not sure about this - the order of annotations shouldn't matter - maybe the best thing is to ignore the JsonScalaEnumeration annotation if there is a Schema annotation - could you provide a case class that could be used in a test case?

Reply:
Added 3 tests for each case (annotated enums, bigint and bigdecimal)